web/app/components/project/jira-widget.hbs

```diff
@@ -1,232 +1,213 @@
-<div
-  class="jira-widget relative flex h-[36px] w-full max-w-3xl items-center"
-  ...attributes
->
+{{#unless (and (not this.issue) (not @isLoading) @isReadOnly)}}
```

Comment:
Style: this logic might be more readable if it was converted to a getter?

Reply:
Yeah the `unless` makes it extra confusing.

I'll wrap this change into #523, because I've already moved the check into the parent component to be used in an  `animated-if` statement. It'll soon be

```
 get jiraWidgetIsShown() {
    return (
      this.projectIsActive || this.jiraIssue || this.loadJiraIssue.isRunning
    );
  }
```